### DISCUSSION TITLE
Why Json field return texte format ?

### DISCUSSION THREAD

**Discussion by Graciasjr** [pocketbase/pocketbase#2628] [Q&A] [answered]
Hi, I have a little problem with the json type fields. the problem is that when rendering the data, i get the response in text format and not in json. I would have liked to parse the response but it is impossible in the context where I work. 
Is ont normal?

**Comment by ganigeorgiev** [comment, marked as answer]
If the JSON field value is not a valid JSON it will be casted to a string. Make sure that the json value you are submitting is properly formatted JSON (eg. the identifiers have double quotes, there is no trailing comma, etc.)

**Reply by Graciasjr** [reply]
just a suggestion 
wouldn't it be better to add a json validation feature in the json field or maybe a json parser in order to transform the misformatted Json

**Reply by ganigeorgiev** [reply]
Plain strings are valid JSON values. We need the string casting in order to seamlessly support both `application/json` and `multipart/form-data` requests. There is also a note in the Admin UI with the performed normalizations:

![screenshot](https://i.imgur.com/xLwroCD.png)

JSON schema validation may be added in the future as part of https://github.com/pocketbase/pocketbase/issues/1154, but for now remains a low priority.

**Reply by Graciasjr** [reply]
Okay i understand . 
Thanks You

**Comment by Graciasjr** [comment]
yes I think that's it, I just saw in this discussion
https://github.com/pocketbase/pocketbase/discussions/2595#discussioncomment-6031911 
that I had to put the quotes, I think that's where my problem comes from
